### PR TITLE
Nerfs King

### PR DESCRIPTION
**Pull request by Alterist**
King is too powerful, this make that not the case

## About The Pull Request

Makes Grav Crush not crush ear drums

## Changelog
:cl:
soundadd: Grav Crush volume has been quieted by around 10dB
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/xenomorph/castes/king/abilities_king.dm [hunk 1]
```diff
@@ -63,7 +63,7 @@
 /datum/action/xeno_action/activable/gravity_crush/use_ability(atom/A)
 	owner.face_atom(A) //Face towards the target so we don't look silly
 	var/list/turfs = RANGE_TURFS(1, A)
-	playsound(A, 'sound/effects/bomb_fall.ogg', 75, FALSE)
+	playsound(A, 'sound/effects/bomb_fall.ogg', 50, FALSE)
```
Comment (BraveMole) [inline code/modules/mob/living/carbon/xenomorph/castes/king/abilities_king.dm:66]: this doesn't do anything, volume is bugged. You have to edit the file yourself

**Review by BraveMole — CHANGES_REQUESTED** [review]

**Comment by Alterist** [comment]
oh

File: code/modules/mob/living/carbon/xenomorph/castes/king/abilities_king.dm (hunk 1 above)
Comment (lbnesquik) [reply code/modules/mob/living/carbon/xenomorph/castes/king/abilities_king.dm:66]: Since when is it bugged?

**Comment by TiviPlus** [comment]
Fix your changelog

**Comment by Alterist** [comment]
what should it be then?

**Comment by Ieteihowmee** [comment]
Agree.

**Pull request merged**